### ISSUE TITLE
Show `RO:0004029` associations from Mondo in Phenotype associations section

### ISSUE THREAD

**Issue by kevinschaper** [monarch-initiative/monarch-app#684]
We disease to phenotype associations in Mondo that we're not showing on the site right now, because they're not getting the right category and predicate. 

examples:
|      category       |    subject    |         subject_label          |     predicate      |         object_label         |   object   |
|---------------------|---------------|--------------------------------|--------------------|------------------------------|------------|
| biolink:Association | MONDO:0000986 | pleurisy                       | biolink:related_to | Pneumonia                    | HP:0002090 |
| biolink:Association | MONDO:0000986 | pleurisy                       | biolink:related_to | Pleuritis                    | HP:0002102 |
| biolink:Association | MONDO:0000986 | pleurisy                       | biolink:related_to | Systemic lupus erythematosus | HP:0002725 |
| biolink:Association | MONDO:0000986 | pleurisy                       | biolink:related_to | Pulmonary embolism           | HP:0002204 |
| biolink:Association | MONDO:0000321 | typhoidal tularemia            | biolink:related_to | Weight loss                  | HP:0001824 |
| biolink:Association | MONDO:0000333 | early congenital syphilis      | biolink:related_to | Seizure                      | HP:0001250 |
| biolink:Association | MONDO:0000333 | early congenital syphilis      | biolink:related_to | Hepatosplenomegaly           | HP:0001433 |
| biolink:Association | MONDO:0000463 | Ochoa syndrome                 | biolink:related_to | Neurogenic bladder           | HP:0000011 |
| biolink:Association | MONDO:0000490 | glomerulosclerosis             | biolink:related_to | Glomerular sclerosis         | HP:0000096 |
| biolink:Association | MONDO:0000875 | adult acute monocytic leukemia | biolink:related_to | Adult onset                  | HP:0003581 |

Ideally, these should wind up with:
```
category: biolink:DiseaseToPhenotypicFeatureAssociation
predicate: biolink:has_phenotype
```

I created a [biolink PR](https://github.com/biolink/biolink-model/pull/1495) that should take care of the predicate (assuming kgx will use the narrow mapping?), but it's important to also make sure we populate the category with `biolink:DiseaseToPhenotypicFeatureAssociation` - I don't know how practical is to populate categories based on predicate + domain & range?   What do you think @caufieldjh

**Comment by caufieldjh** [comment]
Looking into it. To show these in the Monarch interface, is it enough to just have the has_phenotype predicate? I'm guessing it is not

**Comment by kevinschaper** [comment]
The association category also needs to be DiseaseToPhenotypicFeatureAssociation, which might need some hardcoding somewhere.

**Comment by caufieldjh** [comment]
This is partially fixed as of the 20240606 KG-Phenio build (that one has the correct DiseaseToPhenotypicFeatureAssociation categories, but still has `related_to` for predicates.
Both the correct category and predicate should be present for all MONDO to HP edges in the in-progress 20240607 KG-Phenio build

**Comment by kevinschaper** [comment]
Looking good on monarchinitiative.org! 
<img width="1000" alt="Screenshot 2024-06-12 at 12 20 28 PM" src="https://github.com/monarch-initiative/monarch-app/assets/4535019/bd8f10e2-7269-46c3-8636-72913d0760aa">
